### PR TITLE
Implement of Criteria

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
Guys, I finished the implementation of criteria in the PdfToImage class.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/PdfToImage.java [hunk 1]
```diff
@@ -25,18 +25,21 @@
  * @author Josue Rodriguez
  * @version 1.0
  */
-public class PdfToImage extends CriteriaPdfToImage {
-
+public class PdfToImage implements IConvert{
+    CriteriaPdfToImage criterio;
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/PdfToImage.java]: Please change "criterio" to "criterion"

File: src/main/java/com/fundation/webservice/model/PdfToImage.java [hunk 2]
```diff
@@ -25,18 +25,21 @@
...
-                BufferedImage bim = pdfRenderer.renderImageWithDPI(page, getDpi(), ImageType.valueOf(getFormatColor()));
-                String fileName = getDestPath() + getName() + page + getExt();
-                ImageIOUtil.writeImage(bim, fileName, getDpi());
+                BufferedImage bim = pdfRenderer.renderImageWithDPI(page, criterio.getDpi(), ImageType.valueOf(criterio.getFormatColor()));
+                String fileName = criterio.getDestPath() + criterio.getName() + page + criterio.getExt();
+                ImageIOUtil.writeImage(bim, fileName, criterio.getDpi());
             }
         } catch (IOException e) {
-            System.err.println("Exception while trying to create pdf document - " + e);
+
```
Comment (MadayAlcala) [inline src/main/java/com/fundation/webservice/model/PdfToImage.java:42]: Is this space neccesary?

Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/PdfToImage.java:42]: eliminate this space please

Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/PdfToImage.java:42]: ok

**Comment by LimbertVargas** [comment]
it's all ok, good Job papu

**Pull request merged**